schema/index.graphql

```diff
@@ -80,6 +80,7 @@ enum ReservationStatus {
 enum RelatedProductType {
   ACCESSORY
   ALTERNATIVE
+  SUCCESSOR
```

Comment:
Common case especially in B2B commerce is the deprecation of a product, which might have a successor. 
Successors are a special type of alternatives, so it might make sense to handle them different.